samples/userspace/syscall_perf/sample.yaml

```diff
@@ -0,0 +1,3 @@
+sample:
```

Comment:
this sample only works on Risc-V correct?
I think you will need some filtering here so sanitycheck doesn't try to run this on other arches.

Reply:
I don't know if it could work for other arch. By security, I will specify riscv arch as you suggest.

Reply:
Done